crates/proof-of-sql/src/base/database/column_repetition_operation.rs

```diff
@@ -120,19 +136,40 @@ impl RepetitionOp for ColumnRepeatOp {
     fn op<T: Clone>(column: &[T], n: usize) -> impl Iterator<Item = T> {
         repeat_slice(column, n)
     }
+
+    fn op_fixed_size_binary(col_bytes: &[u8], width: usize, n: usize) -> Vec<u8> {
+        let rows: Vec<_> = col_bytes.chunks_exact(width).collect();
+        let mut out = Vec::with_capacity(rows.len() * width * n);
```

Comment:
Please don't create temporary `Vec`s. This is inefficient and unneeded.
Please avoid mutability.